lldb/include/lldb/API/SBLineEntry.h

```diff
@@ -29,6 +29,9 @@ class LLDB_API SBLineEntry {
 
   lldb::SBAddress GetEndAddress() const;
 
+  lldb::SBAddress
+  GetSameLineContiguousAddressRangeEnd(bool include_inlined_functions) const;
+
```

Comment:
We don't need this API right? We spoke about using existing APIs for this. I believe we can find the index the SBLineEntry in the line table using:
```
uint32_t SBCompileUnit::FindLineEntryIndex(lldb::SBLineEntry &line_entry, bool exact = false) const;
```
And then you can use:
```
lldb::SBLineEntry SBCompileUnit::GetLineEntryAtIndex(uint32_t idx) const;
```
With the next index until the line doesn't match.

Reply:
@clayborg, right, I know I can do that but I did not use it for two reasons:
1. That seems to be what `GetSameLineContiguousAddressRangeEnd` API is doing internally so I prefer to reuse the API than implement myself.
2. This seems to depend on the dwarf line table encoding which may not work for other platforms like PDB? Hiding behind `GetSameLineContiguousAddressRangeEnd` makes the code more resilient? 

Anyway, if you still prefer us calling "GetLineEntryAtIndex" repeat, I will change it.

Reply:
I take a second look at `GetSameLineContiguousAddressRangeEnd` implementation, it seems to take care of compiler generate code and inline functions which I find it is tedious to duplicate the implementation in lldb-dap: 
https://github.com/llvm/llvm-project/blob/a6d932bca8875198fbf34564cda8a8d1640cdcbc/lldb/source/Symbol/LineEntry.cpp#L202-L233

So I think it makes more sense to reuse it. Please take a further review. Thanks.